v1/api/swagger/paths/allocation@server@{siteID}.yaml

```diff
@@ -0,0 +1,24 @@
+get:
+  tags:
+    - allocation
+  summary: Get allocation for servers 
+  description: Returns allocation information, for each server instance type used by each PCE service.
+  operationId: allocation_getBySite
+  parameters:
+    - name: siteID
+      in: path
+      description: site ID
+      required: false
```

Comment:
What is returned if site ID is not specified?

Reply:
Will add text about that